### PR TITLE
adding antlrv4 orbit bundle

### PR DESCRIPTION
**Pull request by tishan89**
Adding ANTLR v4 runtime orbit bundle needed for WSO2 CEP 4.0.0 M3. Please review and merge.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,7 +58,8 @@
         <module>cassandra-thrift/1.2.13.wso2v4</module>
         <module>bsf/3.0.0.wso2v4</module>
         <module>lucene/4.10.3-wso2v1</module>
-   	<module>rabbit-mq/3.4.4.wso2v1</module>
+        <module>rabbit-mq/3.4.4.wso2v1</module>
+        <module>antlr-runtime/4.5.wso2v1</module>
```
Comment (arunasujith) [inline pom.xml]: Revert this, since we not adding modules to parent pom.

Comment (tishan89) [inline pom.xml]: Will revert and send a new PR

File: antlr-runtime/4.5.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>1.4.0</version>
```
Comment (arunasujith) [inline antlr-runtime/4.5.wso2v1/pom.xml]: Update the maven-bundle-plugin version

File: antlr-runtime/4.5.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.antlr.v4.runtime.*;version=4.5
```
Comment (arunasujith) [inline antlr-runtime/4.5.wso2v1/pom.xml]: Can you please check whether are there any imports?

File: antlr-runtime/4.5.wso2v1/pom.xml (hunk 2 above)
Comment (tishan89) [inline antlr-runtime/4.5.wso2v1/pom.xml]: Noted. Should I update it to latest version(2.5.3)?

**Pull request closed without merging**